README.md

```diff
@@ -14,19 +14,13 @@
 $ yarn
 ```
 
-2. Log into Shopify store via CLI:
+2. Start the Shopify dev server.
 
 ```
-$ shopify login --store shopify-store
+$ yarn theme:start --store shopify-store
 ```
 
-3. Start the Shopify dev server.
-
-```
-$ yarn theme:start
```

Comment:
I want to understand why this was removed. It seems like we'd want the actual command to use in the README, yes?

Reply:
gotcha! so out of the loop. merging.